src/Trixi.jl

```diff
@@ -79,11 +79,14 @@ export flux_central, flux_lax_friedrichs, flux_hll, flux_upwind,
 
 export initial_condition_constant,
        initial_condition_gauss,
+       initial_condition_density_wave,
+       initial_condition_isentropic_vortex,
+       initial_condition_khi,
        initial_condition_weak_blast_wave, initial_condition_blast_wave,
-       initial_condition_khi, initial_condition_medium_sedov_blast_wave,
-       initial_condition_isentropic_vortex, initial_condition_blob,
-       initial_condition_density_wave, initial_condition_sedov_blast_wave,
-       initial_condition_orszag_tang, initial_condition_rotor
+       initial_condition_sedov_blast_wave,
```

Comment:
```suggestion
       initial_condition_sedov_blast_wave, initial_condition_medium_sedov_blast_wave,
```
Do you really want to remove the `initial_condition_medium_sedov_blast_wave`?

Reply:
No, thanks for catching this. Merge FUBAR :-/

Reply:
Fixed in f50a4c2